### DISCUSSION TITLE
v0.7.0

### DISCUSSION THREAD

**Discussion by Blinue** [Blinue/Magpie#201] [Announcements]
v0.7 含有大量的新功能和改进，这里只列出最重要的。注意，和 v0.7.0 Preview1 相比此版本没有可以改变 FSR 锐度的滑块，想修改的用户请参考[自定义缩放配置](https://github.com/Blinue/Magpie/wiki/%E8%87%AA%E5%AE%9A%E4%B9%89%E7%BC%A9%E6%94%BE%E9%85%8D%E7%BD%AE)。

感谢 @palxex 和 @hooke007，在开发过程中他们花了大量时间测试新功能。

v0.7 更新内容

1. 和 v0.6.1 相比性能大幅提升，且资源占用更少
2. 大幅提高 GDI 捕获模式的性能，添加新的捕获模式 DwmSharedSurface。它们均可以感知 DPI 缩放
2. 所有效果均已移植为 MagpieFX，现在它们更快，视觉效果更好
3. 已移植 Anime4K v4 的多种着色器 #87 
4. 已移植 xBRZ 族，Pixellate 和 Sharp Bilinear，它们适合像素游戏的缩放 #62 #103 
5. 支持不绘制光标和缩放光标
6. 支持以管理员权限启动 #64 
7. 已移植 FXAA 和 SMAA #74 
8. 支持关闭垂直同步和指定帧率
9. 支持 Win11 中在缩放时禁用窗口圆角
10. 很多其他的改进和优化

注意：
1. 不再支持 SSimSuperRes，Anime4K DarkLines 和 Anime4K ThinLines
2. ScaleModels.json 和 v0.6.1 不兼容

最后推荐一个 Fork：[Z-Dante/Magpie](https://github.com/Z-Dante/Magpie)，它聚焦于视觉小说游戏相关的改进

<hr /><em>This discussion was created from the release <a href='https://github.com/Blinue/Magpie/releases/tag/v0.7.0'>v0.7.0</a>.</em>

**Comment by hooke007** [comment]
贴一个我自己用的自定义缩放示例以供参考，方便快速测试单项着色器的表现
https://gist.github.com/hooke007/818ecc88f18e229bca743b7ae48947ad

**Comment by login404** [comment]
v0.7.0最小化跟关闭到托盘
都会导致任务栏和托盘的图标消失
但进程仍然在运行

**Reply by hooke007** [reply]
如果可稳定复现，欢迎提交 [issue](https://github.com/Blinue/Magpie/issues/new/choose) 报错

**Reply by palxex** [reply]
同样遇到过这个问题，但没找到过稳定复现办法。
建议不要把最小化离开任务栏进托盘做成默认的，这个跟windows用户的直觉并不相符。

**Reply by Blinue** [reply]
旧版本 win10 可能出现这个问题。解决方案 👉 https://github.com/Blinue/Magpie/issues/203#issuecomment-969882957

**Comment by ghost** [comment]
despite the fact that i miss the FSR slider and dont get it why it is removed this updates contains lots of improvements.. but i cant see any difference when i change the default sharpening value (0.87) between 0.01 and 0.99 for FSR in ScaleModels.json file. is this really working ? anyway i cant wait to see v0.8 with UWP games support. thnx

**Reply by hooke007** [reply]
Don't be afraid that sharpness over 1 would break. And CAS is sensitive to noise.

sharpness=0.01
![Snipaste_2021-11-17_03-24-27](https://user-images.githubusercontent.com/41094733/142052317-e8d35a4a-836c-4888-88a3-81b774be4f08.jpg)

sharpness=1.5
![Snipaste_2021-11-17_03-37-55](https://user-images.githubusercontent.com/41094733/142053825-e624f093-3688-42d5-8e3d-6e1df520ad7c.jpg)

**Reply by ghost** [reply]
uhh thanks for the info..im gonna try it now

**Comment by MorderczkazPolski12** [comment]
Hello

what is the difference between FSR and FSRCNNX?

**Reply by hooke007** [reply]
fsrcnnx is developed from srcnn, which is high-quality but not designed for 3d videogames.

http://mmlab.ie.cuhk.edu.hk/projects/FSRCNN.html

**Reply by MorderczkazPolski12** [reply]
ok thanks

**Comment by frdoma** [comment]
请问一下，Magpie里的ACNet的降噪强度对应ACNet GLSL的1-3的什么级别？经过测试比Anime4K_Upscale_Denoise_L的降噪效果还要强一些，如果ACNet的降噪强度能再降低一些或许更好。

**Reply by Blinue** [reply]
Magpie 中的 ACNet 是无降噪版本

**Reply by frdoma** [reply]
> Magpie 中的 ACNet 是无降噪版本

好的，另外0.6版的Magpie里面FSR是有降噪选项的，0.7版的没有了。原来FSR里的这个和Anime4K单独提供的降噪算法一样都是通用降噪算法吗？

**Reply by Blinue** [reply]
FSR 中的降噪算法不通用，v0.7 中的 FSR 始终是降噪的

**Reply by frdoma** [reply]
> FSR 中的降噪算法不通用，v0.7 中的 FSR 始终是降噪的

0.6版FSR的降噪在FsrRcas里可选开启关闭，0.7版集成在FSR_EASU里始终开启了是吧，了解了。

**Comment by Lisy8114** [comment]
開啟FSR遊玩BATTLEFIELD 2042雖然張數顯示提升但畫面卻有明顯卡頓，想問如何處理

**Reply by Blinue** [reply]
1. 试试更换捕获模式
2. 试试在设置中打开“禁用 DirectFlip”
